src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexFiltering.cpp

```diff
@@ -51,19 +51,42 @@ using namespace boost::math;
 namespace OpenMS
 {
 
-  MultiplexFiltering::MultiplexFiltering(const MSExperiment& exp_picked, const std::vector<MultiplexIsotopicPeakPattern>& patterns,
+  MultiplexFiltering::MultiplexFiltering(const MSExperiment& exp_centroided, const std::vector<MultiplexIsotopicPeakPattern>& patterns,
                                          int isotopes_per_peptide_min, int isotopes_per_peptide_max, double intensity_cutoff, double rt_band,
                                          double mz_tolerance, bool mz_tolerance_unit, double peptide_similarity, double averagine_similarity,
                                          double averagine_similarity_scaling, String averagine_type) :
-  exp_picked_(exp_picked), patterns_(patterns), isotopes_per_peptide_min_(isotopes_per_peptide_min), isotopes_per_peptide_max_(isotopes_per_peptide_max),
+  patterns_(patterns), isotopes_per_peptide_min_(isotopes_per_peptide_min), isotopes_per_peptide_max_(isotopes_per_peptide_max),
   intensity_cutoff_(intensity_cutoff), rt_band_(rt_band), mz_tolerance_(mz_tolerance), mz_tolerance_unit_in_ppm_(mz_tolerance_unit),
   peptide_similarity_(peptide_similarity), averagine_similarity_(averagine_similarity),
   averagine_similarity_scaling_(averagine_similarity_scaling), averagine_type_(averagine_type)
   {
+    // initialise experiment exp_centroided_
```

Comment:
to save memory and speed up calculations, I suggest here:

```
exp_centroided_.reserve(exp_centroided.getNrSpectra())
```

Reply:
[Done.](https://github.com/OpenMS/OpenMS/pull/3768/files#diff-d07005aa96a93e57990ba6d481c8425eR65)